### PR TITLE
Generated SDK

### PR DESCRIPTION
**Pull request by ishanc008**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — CHANGES_REQUESTED** [review]
To start, please ensure that you're following the Azure SDK onboarding process and that your account has joined both the Microsoft and Azure organizations, and the membership is public on your profile.

Given the name of the RP used and link to the ReadMe rather than an API spec, I'm unsure if this was a legitimate mistake or a spam PR.   Blocking until we have clarity.

File: eng/scripts/GenerateAndBuildLib.ps1 [hunk 1]
```diff
@@ -0,0 +1,886 @@
+#Requires -Version 7.0
```
Comment (ArthurMa1978) [inline eng/scripts/GenerateAndBuildLib.ps1:1]: remove this

**Comment by ArthurMa1978** [comment]
@ishanc008, here is already have the [sdk](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/networkfunction/Azure.ResourceManager.NetworkFunction) for network function, please upgrade the existing one instead of create a new one

File: eng/scripts/GenerateAndBuildLib.ps1 (hunk 1 above)
Comment (ishanc008) [reply eng/scripts/GenerateAndBuildLib.ps1:1]: Hi Arthur, I didn't add this line of code, it came in from the forked repo azure-sdk-for-net.

**Comment by ishanc008** [comment]
> To start, please ensure that you're following the Azure SDK onboarding process and that your account has joined both the Microsoft and Azure organizations, and the membership is public on your profile.
> 
> Given the name of the RP used and link to the ReadMe rather than an API spec, I'm unsure if this was a legitimate mistake or a spam PR. Blocking until we have clarity.

Hi @jsquire I have followed all the steps from this documentation [doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process#sdk-generation-from-scratch). In this doc it was written that we have to provide link to readme file only from the public [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo. So I took the permalink of my [readme file](https://github.com/Azure/azure-rest-api-specs/blob/85d0d6c98a47a6b6c2c1684cf94cc8e089c3c4ab/specification/mpcnetworkfunction/resource-manager/readme.md) and copied it in my autorest.md. Can you please reopen this PR.
Thanks

**Comment by jsquire** [comment]
> Hi @jsquire I have followed all the steps from this documentation [doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process#sdk-generation-from-scratch). In this doc it was written that we have to provide link to readme file only from the public [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo. So I took the permalink of my [readme file](https://github.com/Azure/azure-rest-api-specs/blob/85d0d6c98a47a6b6c2c1684cf94cc8e089c3c4ab/specification/mpcnetworkfunction/resource-manager/readme.md) and copied it in my autorest.md. Can you please reopen this PR.

Please reach out to the Azure SDK onboarding team and follow their guidance.   Your account is not in the correct GitHub organizations, which we'll need to sort out before I remove the block.

**Comment by ArthurMa1978** [comment]
Please update this pr with the new namespace https://github.com/Azure/azure-sdk-pr/issues/1638

**Pull request closed without merging**